package.json

```diff
@@ -67,12 +67,14 @@
 		"csv": "~5.3.2",
 		"csv-stringify": "~5.6.5",
 		"d3": "~7.8.5",
+		"dompurify": "~2.4.0",
```

Comment:
The latest is 3.25 and 2.4.0 is from 8/14/22 which seems fairly old. Could a newer version be used?

The two new packages are about 4 MB in totaL. This isn't trivial in size. Are the full packages of both needed? Since it is only on the server does it get added to the client-side or just the server-side bundle?